Makefile

```diff
@@ -1,48 +1,67 @@
-JSONNET = https://github.com/google/jsonnet/releases/download/v0.17.0/jsonnet-bin-v0.17.0-linux.tar.gz
+JB = https://github.com/jsonnet-bundler/jsonnet-bundler/releases/latest/download/jb-linux-amd64
 BINDIR = bin
 TEMPLATESDIR = templates
 OUTPUTDIR = rendered
 ALLDIRS = $(BINDIR) $(OUTPUTDIR)
 SYNCER_IMG_TAG ?= quay.io/cloud-bulldozer/dittybopper-syncer:latest
 PLATFORM = linux/amd64,linux/arm64,linux/ppc64le,linux/s390x
 
-# Get all templates at $(TEMPLATESDIR)
-TEMPLATES := $(wildcard $(TEMPLATESDIR)/**/*.jsonnet)
+ifeq ($(filter v2,$(MAKECMDGOALS)),v2)
+  	# Set variables and instructions for v2
+  	TEMPLATES := $(wildcard $(TEMPLATESDIR)/**/*-v2.jsonnet)
+	LIBRARY_PATH := $(TEMPLATESDIR)/vendor
+	JSONNET := https://github.com/google/go-jsonnet/releases/download/v0.20.0/go-jsonnet_0.20.0_Linux_x86_64.tar.gz
```

Comment:
Do we want to var this version out some to allow easier swapping of `go-jsonnet` ?

Reply:
Took this variable to the top as it is compatible with both old and new dashboards code. I think an extra var is not required now as it a single line change.